roles/common/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+deployment_type: awx
+kind: '{{ deployment_type | upper }}'
+api_version: '{{ deployment_type }}.ansible.com/v1beta1'
```

Comment:
If we move those variables from the installer role then we can remove them from that role to avoid duplicate declaration ?

Reply:
I left them in in case people wanted to run the installer role directly, like in this PR - https://github.com/ansible/awx-operator/pull/1016

I suppose now that we have `playbooks/awx.yml`, that might be a good place to put global variables.